### ISSUE TITLE
[Quest]Extinguishing the idol 

### ISSUE THREAD

**Issue by Nosere** [Atlantiss/BugTracker#3671]
http://www.wowhead.com/quest=3525/extinguishing-the-idol

The quest does not complete after the last wave.

**Comment by Quban** [comment]
it still bugs? coz i've tested it many times and it works every time

**Comment by RastaTinu** [comment]
Quest still doesn't work.

**Comment by TranceAccount** [comment]
Bugged.

**Comment by aendry** [comment]
Yep, still broken

**Comment by podatekvat** [comment]
Yes, as of v5536 it's STILL not working. NPC endlessly casting spell on idol even after killing the boss after 5min timer.
